### PR TITLE
Update website copy as part of TLC work

### PR DESCRIPTION
**Pull request by michelle-rago**
# Pull request summary
- Update text on 18F site according to https://github.com/18F/TLC-crew/issues/317

[View these changes](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/tlc-copy-07-2023/)

### CODE REVIEW

**Comment by michelle-rago** [comment]
@AnitaYCheng Maybe you can figure out why "Get in touch" is floating to the right on https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/tlc-copy-07-2023/. 😸

**Comment by AnitaYCheng** [comment]
> @AnitaYCheng Maybe you can figure out why "Get in touch" is floating to the right on https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/tlc-copy-07-2023/. 😸

Turns out the height set for the hero section was too short - had to add some custom CSS code to increase it. Apparently the text for the hero section is longer than the previous version.

File: _sass/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -108,3 +108,10 @@ a {
 footer {
   order: unset;
 }
+
+// increase height for hero section
+@media (min-width: 640px) {
+  .hero__container {
+      max-height: 550px;
+  }
+}
```
Comment (jskinne3) [inline _sass/_uswds-theme-custom-styles.scss:117]: It's not important but the convention is to use a newline at the end of a file

**Review by jskinne3 — APPROVED** [review]

File: _sass/_uswds-theme-custom-styles.scss (hunk 1 above)
Comment (AnitaYCheng) [reply _sass/_uswds-theme-custom-styles.scss:133]: What do you mean by "using a newline at the end of a file?" What would that look like here?

Comment (jskinne3) [reply _sass/_uswds-theme-custom-styles.scss:133]:
```suggestion
}

```
Just pressing return / enter at the end of the file, like this

**Pull request merged**